modules/fb_instant_articles_views/src/Plugin/views/filter/ValidFacebookInstantArticles.php

```diff
@@ -64,68 +66,70 @@ public function init(ViewExecutable $view, DisplayPluginBase $display, array &$o
 
   /**
    * {@inheritdoc}
-   *
-   *
    */
   public function query() {
-
     $this->enabledNodeBundlesSetValues();
     parent::query();
   }
 
   /**
-   * Set the values for this filter to all node bundles that implement custom settings for the view mode
+   * Set the values for this filter.
    *
-   * This uses an unreliable method for detecting custom settings for node bundles.  The method is to check for a config
-   * Entity for the entity display settings, for the bundle/view mode.  Essentially this looks for a yml file/record
-   * for the custom settings, and one is found, the bundle is considered FIA active.
+   * This applies to all node bundles that implement custom settings for the
+   * view mode.
    *
+   * This uses an unreliable method for detecting custom settings for node
+   * bundles.  The method is to check for a config entity for the entity display
+   * settings, for the bundle/view mode.  Essentially this looks for a yml
+   * file/record for the custom settings, and one is found, the bundle is
+   * considered FIA active.
    */
   protected function enabledNodeBundlesSetValues() {
-    /**
-     * @var \Drupal\Core\Entity\EntityTypeBundleInfo $entityBundleInfo
+    /*
+     * @var \Drupal\Core\Entity\EntityTypeBundleInfo $entity_bundle_info
      *  entity_type.bundle.info
      */
-    $entityBundleInfo = \Drupal::service('entity_type.bundle.info');
+    $entity_bundle_info = \Drupal::service('entity_type.bundle.info');
 
-    /**
-     * @var \Drupal\Core\Entity\EntityStorageInterface $entityStorage
+    /*
+     * @var \Drupal\Core\Entity\EntityStorageInterface $entity_storage
      *   View Mode entity storage handler
      */
-    $entityStorage = \Drupal::service('entity_type.manager')->getStorage('entity_view_display');
+    $entity_storage = \Drupal::service('entity_type.manager')->getStorage('entity_view_display');
 
-    /**
+    /*
      * @var string[] nodeTypes
      *   an array of node types that implement our custom view mode
      */
-    $nodeTypes = [];
-    foreach($entityBundleInfo->getBundleInfo('node') as $id=>$bundle) {
-      /**
-       * @var string $viewModeId
+    $node_types = [];
+    foreach ($entity_bundle_info->getBundleInfo('node') as $id => $bundle) {
+
+      /*
+       * @var string $view_mode_id
        *   the string id for the view mode entity
        */
-      $viewModeId = 'node.'.$id.'.'.static::FIA_VIEW_MODE;
+      $view_mode_id = 'node.' . $id . '.' . static::FIA_VIEW_MODE;
 
-      /**
-       * @var \Drupal\Core\Entity\EntityInterface|null $viewMode
+      /*
+       * @var \Drupal\Core\Entity\EntityInterface|null $view_mode
        *   Config entity for the view mode, if it exists
        */
-      $viewMode = $entityStorage->load($viewModeId);
+      $view_mode = $entity_storage->load($view_mode_id);
 
-      if ($viewMode instanceof EntityInterface) {
-        $nodeTypes[$id] = $id;
+      if ($view_mode instanceof EntityInterface) {
```

Comment:
Sounds logical, but I'm not able to make that work.  Ended up with `node_field_data.type = ''` in the query.